### PR TITLE
allows anyone to post a grant without having to pay gas fees

### PR DESCRIPTION
**Pull request by owocki**
##### Description

allows anyone to post a grant without having to pay gas fees

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/6817

##### Testing

tested locally

### CODE REVIEW

**Comment by danlipert** [comment]
@owocki all we'd have to do is change the contract version in the `new-whitelabel.html` template

File: app/grants/models.py [hunk 1]
```diff
@@ -324,6 +324,16 @@ def updateActiveSubscriptions(self):
         self.activeSubscriptions = handles
 
 
+    @property
+    def recurring_funding_supported(self):
+        return self.contract_version < 2:
```
Comment (danlipert) [inline app/grants/models.py:329]: Saw travis was failing on this error - what is this property used for?

**Comment by apbendi** [comment]
Hey @owocki, if this is merged, would we remove the `fund.html` & `fund.js` files from the grants round 6 integration branch, along with the link on the details page to do recurring funding? Or do we still leave that for legacy grants that support it?

File: app/grants/models.py (hunk 1 above)
Comment (owocki) [reply app/grants/models.py:329]: ill probably thread it through to the recurring fund template if we keep it

**Comment by owocki** [comment]
@apbendi lets see if vitalik wants us to keep recurring contributions or not

**Comment by owocki** [comment]
ok just dropped this over to him 

```
Hey Vitalik,

We are thinking of completely removing recurring contributions this round + focusing solely on the "add all your grants from last round to cart" functionality to provide sustainability to funders.

Do you have any objections to this path?

Reasons why:
- People dont seem to understand/trust our ability to debit their wallets on an ongoing basis.
- It was barely used last round.  Only a few pct of contributions were recurring.
- We are moving towards "one tx checkout of many grants", which feels important for a world where gas prices are high. We cant do recurring contributions in one tx checkout in bulk checkout.
- We think we can make up the round by reminding people what they contributed to last time + haveing a "one click add all to cart" button.

Hey Vitalik,

We are thinking of completely removing recurring contributions this round + focusing solely on the "add all your grants from last round to cart" functionality to provide sustainability to funders.

Do you have any objections to this path?

Reasons why:
- People dont seem to understand/trust our ability to debit their wallets on an ongoing basis.
- It was barely used last round.  Only a few pct of contributions were recurring.
- We are moving towards "one tx checkout of many grants", which feels important for a world where gas prices are high. We cant do recurring contributions in one tx checkout in bulk checkout.
- We think we can make up the round by reminding people what they contributed to last time + haveing a "one click add all to cart" button.

```

**Comment by owocki** [comment]
@apbendi vitalik gave us the 👍 . lets kill fund.html and fund.js

**Comment by owocki** [comment]
@apbendi lmk if you want to nuke fund.html / fund.js. happy to do the honors myself too!

**Comment by apbendi** [comment]
Hi Kevin, just took a quick look. I'll defer to you, if you don't mind
tackling it. There was one place where the fund template was being rendered
that I wasn't sure what it was for. You'll probably bang it out much
quicker than I would.

**Comment by owocki** [comment]
ok ive taken care of that now!

**Comment by owocki** [comment]
cherry-picked over

```
kevinowocki@local /Users/kevinowocki/Sites/gitcoin/web~ % git cherry-pick 617b834f3a2e13dab0806687896448ca3a7065ad
[grants-round-6-integration ded7bfa00] allows anyone to post a grant without having to pay gas fees
 Date: Wed Jun 10 16:26:49 2020 -0600
 4 files changed, 35 insertions(+), 151 deletions(-)
kevinowocki@local /Users/kevinowocki/Sites/gitcoin/web~ % git cherry-pick 1d1e66dd8294b895d37713a7b846bac5da02971c
[grants-round-6-integration e76f9b430] method that says iff recurring funding is supported
 Date: Wed Jun 10 16:30:50 2020 -0600
 1 file changed, 4 insertions(+)
```

File: app/grants/models.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/models.py:329]: @owocki this got into grants-round-6 !
I've removed the `:` over here https://github.com/gitcoinco/web/pull/6807/commits/3c9431c83eb406e436d6119f4b6478ae3a645e84

Comment (owocki) [reply app/grants/models.py:329]: derp, my bad

**Pull request closed without merging**